### ISSUE TITLE
Users are able to select RKE2 services for cert rotation on K3s clusters

### ISSUE THREAD

**Issue by jameson-mcghee** [rancher/dashboard#6926]
**Setup**
- Rancher version: 2.7-head (Commit ID: db75fbb)
- Browser type & version: Brave: v1.43.93 (Chromium: v105.0.5195.127)

**Describe the bug**
When rotating certificates of individual services on K3s clusters, users are able to select the options _rke2-controller_ and _rke2-server_ which are RKE2 specific

**To Reproduce**

1. Deploy a 3 etcd, 2 control plane, 3 worker nodes K3s cluster
2. Once the cluster comes up Active, navigate to the Cluster Management --> Cluster page for the cluster
3. Click the 3-Dot button for the cluster, and select "Rotate Certificates"
4. Select the option "Rotate an individual service", and expand the dropdown list that appears
- Note that the service options  "rke2-controller" and "rke2-server" are present
5. Select the option "rke2-controller", and then click the "Rotate" button

**Result**
An error message appears in the Header of the cluster, the cluster goes into a permanent "Updating" state, the Control Plane goes into an "Error" state, and an error message is generated on the Control Plane
In addition, numerous errors are observed in the Provisioning Log as well

**Expected Result**
The options _rke2-controller_ and _rke2-server_ are not available for selection on K3s clusters and instead user should have the options _k3s-controller_ and _k3s-server_

**Screenshots**
![image](https://user-images.githubusercontent.com/13294321/191049892-11504dbe-423b-413a-a851-dc7cfd24b723.png)
![image](https://user-images.githubusercontent.com/13294321/191049057-61eccb88-20fe-46d8-9d47-f77b3eec2646.png)

**Comment by gaktive** [comment]
@nwmac dug into the code and this dropdown is hardcoded for RKE2. Down the road, we should see if we can get this list somehow (by looking at the provider), but we may need to just add k3s values.

**Comment by jameson-mcghee** [comment]
I verified that this issue no longer reproduces on `v2.7.0-rc8`. I confirmed that K3s clusters now see the Individual Services `k3s-controller` and `k3s-server` in the Rotate Cluster Certificates list. I also verified that the RKE2 Individual Services `rke2-controller` and `rke2-server` remain in the Rotate Cluster Certificates list for RKE2 clusters. Therefore I am closing this ticket as Done.

K3s:
![image.png](https://zube.io/files/rancher/2027ac9eb826beaad10ad8dd8b3ea0fe-image.png)

RKE2:
![image.png](https://zube.io/files/rancher/315ef1b515ff85accb2ec5bf5e2cdf95-image.png)
